Document new test_command options (#789)
packit/packit.dev: docs/configuration/index.md

This allows commonly used job to be triggered without the need for manual specification. 
For instance, rather than specifying frequently used job via `--identifier` each time, such as `/packit test --identfier my-id`, 
you can configure the following:

```yaml
default_identifier: my-id
```
As a result, invoking `/packit test` will be automatically interpreted as `/packit test --identfier my-id`.

##### default_labels
A list of labels (refer to [`labels`](./configuration/upstream/tests#optional-parameters)) utilised by default when
the `/packit test` comment command is invoked without arguments 
(see [related docs](./configuration/upstream/tests#running-a-group-of-tests-with-the-same-label)).
This allows commonly used job combinations to be triggered without the need for manual specification. 
For instance, rather than specifying frequently used jobs via `--labels` each time, such as `/packit test --labels regression,upgrade`, 
you can configure the following:
```yaml
default_labels:
  - regression
  - upgrade
```
As a result, invoking `/packit test` will be automatically interpreted as `/packit test --labels regression,upgrade`.

:::info

> `default_identifier` and `default_labels` are not supposed to be used simultaneously. You should configure

Reviewer: Shouldn't we be more strict here? What happens if a user configures both?

```suggestion
`default_identifier` and `default_labels` can't be used simultaneously. You can configure
```

Author: well, they can be configured simultaneously, nothing will break, but only one of them will be taken into consideration

> only on of these options.

Reviewer: ```suggestion
only one of these options.
```
